.github/workflows/tests.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
     - run: go version
     - run: go mod download # Not required, used to segregate module download vs test times
     - name: Integration test
-      run: go test -v ./docker_test.go -slow -tags=integration -count 1
+      run: SLOW=1 go test -v ./docker_test.go -tags=integration -count 1
```

Comment:
It might be better to use the opposite semantics, i.e. without you overriding any environment variable all tests should run, no matter if they are slow, integration, etc.

A user can then opt to exclude some tests by setting the relevant env variable overrides.

The only reason why the above solution might be better, is that it eliminates the risk of misconfiguration (forgetting to set an env variable) in our ci workflows, resulting in some tests being skipped inadvertently.

Reply:
I do like the idea, we could have `SLOW` as `true` by default and then we set it to `false` in `make test`.